cmd/apiserver/app/server/options.go

```diff
@@ -41,6 +41,7 @@ type ServiceCatalogServerOptions struct {
 	EtcdOptions *EtcdOptions
 	// TPROptions are options for serving with TPR as the backing store
 	TPROptions *TPROptions
+	StopCh     <-chan struct{}
```

Comment:
Since sending in a nil channel is bad, I had to create one almost like it was before.